app/config/frontendAppConfig.scala

```diff
@@ -68,6 +68,8 @@ trait AppConfig extends ServicesConfig {
   val submitVatReturnBase: String
   val submitVatReturnUrl: String
   val submitVatReturnForm: String => String
+  val host: String
```

Comment:
Could `host` be taken out of the `AppConfig` trait and instead just be a private val in `FrontendAppConfig`? Look like it's only `feedbackUrl` that needs to be accessible outside of here.

Reply:
Good point - I copied the first draft from somewhere that we use the "host" outside of the config so that carried across. I'll get it removed